tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_config_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto";
+
+service TenantIsolationConfigService{
+  rpc GetAllTenantIsolationGroupConfigs(GetAllTenantIsolationGroupConfigsRequest) returns (GetAllTenantIsolationGroupConfigsResponse) {}
+  rpc CreateTenantIsolationGroupConfig(CreateTenantIsolationGroupConfigRequest) returns (CreateTenantIsolationGroupConfigResponse) {}
+  rpc UpdateTenantIsolationGroupConfig(UpdateTenantIsolationGroupConfigRequest) returns (UpdateTenantIsolationGroupConfigResponse) {}
+  rpc DeleteTenantIsolationGroupConfig(DeleteTenantIsolationGroupConfigRequest) returns (DeleteTenantIsolationGroupConfigResponse) {}
+}
+
+message GetAllTenantIsolationGroupConfigsRequest{
+}
+
+message GetAllTenantIsolationGroupConfigsResponse{
+  repeated TenantIsolationGroupConfig configs = 1;
+}
+
+message CreateTenantIsolationGroupConfigRequest{
+  NewTenantIsolationGroupConfig config = 1;
+}
+
+message CreateTenantIsolationGroupConfigResponse{
+  TenantIsolationGroupConfig config = 1;
+}
+
+message UpdateTenantIsolationGroupConfigRequest{
+  TenantIsolationGroupConfig config = 1;
```

Comment:
accept a different item for update than you return. The few lines of code we save by reusing a type is not worth forever coupling the request and response. It would prevent us from ever representing read-only fields, for example.